### ISSUE TITLE
"Show Next Change" no longer works after update to v1.76.0

### ISSUE THREAD

**Issue by monk-time** [microsoft/vscode#175857]
Type: <b>Bug</b>

After updating to v1.76.0 command "Show Next Change" (`alt+f3`) no longer works. My file has uncommited changes that are highlighted in a bar to the left of the file, and when I press the shortcut or trigger the command manually from Command Palette I expect my cursor to move to a line with a change and a panel with a diff to appear (as it used to before the update). But nothing happens. 

![2023-03-02_12-45-12](https://user-images.githubusercontent.com/7759622/222394196-14aef388-b3f8-4fbf-af48-1fdbcbcb512e.gif)

I tried testing this with all extensions disabled and in VSCode Insiders v1.76.0, same there.

VS Code version: Code 1.76.0 (92da9481c0904c6adfe372c12da3b7748d74bdcb, 2023-03-01T10:22:44.506Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4710HQ CPU @ 2.50GHz (8 x 2494)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.92GB (8.21GB free)|
|Process Argv|--crash-reporter-id b0525075-7cdd-471f-b105-651841962233|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
overtype|ada|0.2.0
pascal-language-basics|Ans|0.1.10
vscode-django|bat|1.10.0
vscode-eslint|dba|2.4.0
xml|Dot|2.5.1
gitlens|eam|13.3.1
hoogle-vscode|jca|0.0.7
language-haskell|jus|3.6.0
isort|ms-|2022.8.0
python|ms-|2023.4.0
vscode-pylance|ms-|2023.3.10
jupyter|ms-|2023.2.1000592019
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
remote-wsl|ms-|0.76.1
block-travel|sas|0.0.2
selectline-statusbar|tom|0.0.2
haskero|Van|1.3.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vsccc:30610678
pyindex848:30662994
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
f6dab269:30613381
vscrp:30673768
pythonsymbol12:30671437
6233i204:30672705

```

</details>

**Comment by frypf** [comment]
Same for me, neither the `editor.action.dirtydiff.next` nor `editor.action.dirtydiff.previous` work since upgrading. `workbench.action.editor.nextChange` & `workbench.action.editor.previousChange` seem unaffected, but I want to see the context of changes without having to click the mouse.

**Comment by ahejlsberg** [comment]
I'm having this problem too.

**Comment by ahejlsberg** [comment]
@mjbvz Something seems to be pretty busted in 1.76.0. The Show Next Change (Alt-F3) and Show Previous Change (Shift-Alt-F3) commands do nothing. But worse, the Git: Revert Selected Ranges (Ctrl-K, Ctrl-R) command completely messes up my file.

**Comment by lszomoru** [comment]
I believe that this should be fixed in the latest Stable release,`1.76.2`, so I will optimistically close the issue.
Please reopen it in case you are still able to reproduce the problem with the latest Stable release. Thanks!

**Comment by monk-time** [comment]
@lszomoru Unfortunately cannot confirm, the shortcut and the command are still broken both in 1.76.2 and in 1.77.0-insider. 

I cannot reopen the issue but I hope someone else will.

**Comment by lszomoru** [comment]
@alexr00, could you please take a look and see if this could also be related to your changes? Thank you!

**Comment by kamenminkov** [comment]
Glad to see this fixed; for me however the diff peek that opens seems to always show several lines above the current change. Are there settings that might affect this?

**Comment by alexr00** [comment]
@kamenminkov I'm not aware of any settings that would affect this. Do you see this behavior 100% of the time? If not, do you know if there are any specific steps that cause it to happen?

**Comment by kamenminkov** [comment]
@alexr00 Just found out that it's probably the sticky scroll that's causing this. I disabled all my extensions and still managed to reproduce this. I didn't pay attention at first, but now that I think of it, most of the times it's happened, I've had a bunch of lines showing in the sticky scroll header. I have this in my settings:

```json
  "editor.stickyScroll.enabled": true,
  "editor.stickyScroll.maxLineCount": 10,
```

Tried disabling it (or just lowering the number of lines shown) and now the diff editor shows the difference correctly (albeit a bit towards the bottom).

So maybe the diff editor should either account for the number of sticky scroll lines, or not show the sticky header at all.
